doc/Dashboard.md

```diff
@@ -0,0 +1,82 @@
+"Dashboard" Self-Assessment Summary Block
+=========================================
+
+A "Dashboard" XBlock provides a concise way to summarize a student's answers to
+groups of multiple choice questions.
+
+You configure it like this, by pasting in the `url_name`s of some Problem
+Builder ("Mentoring") blocks:
+![Screen shot of Dashboard XBlock configuration](img/dashboard-configuration.png)
+
+
+And it will then look like this (after a student has submitted their answers):
+![Screen shot of a Dashboard XBlock](img/dashboard-example.png)
+
+
+Color Coding Rules:
+-------------------
+
+Authors can add a list of rules (one per line) that apply colors to the various
+possible student answer values.
+
+Rules are entered into the dashboard configuration "Color Coding Rules", one
+rule per line. The first rule to match a given value will be used to color
+that value.
+
+The simplest rule looks like "3: red". With this line, if the student's answer
+is 3, it will be shown in red in the report. Colors can be specified using any
+valid CSS color value, e.g. "red", "#f00", "#ff0000", "rgb(255,0,0)", etc.
+
+For more advanced rules, you can use an expression in terms of x such as
+"x > 3: blue" or "0 <= x < 5: green" (green if x is greater than or equal to
+zero but less than five).
+
+You can also just specify a color on a line by itself, which will always match
+any value (usually this would be the last line, as a "default" color).
+
+
+Visual Representation
+---------------------
+
+The Dashboard also supports an optional visual representation. This is a
+powerful feature, but setting it up is a bit involved.
+
+The end result is shown below. You can see a diagram, in which a colored arrow
+appears as the student works through the various "Steps". Each "Step" is one
+mentoring block, which contains several multiple choice questions. Based on the
+average value of the student's choices, the step is given a color.
+
+In this example, steps that have not been attempted are transparent, and steps
+that have been attempted are colored in according to the color coding rules.
+
+![Screen shot of visual representation](img/dashboard-visual.png)
+
+To achieve the result shown above requires a set of "stacked" image files (one
+for each step), as well as an overlay image (in this case, the overlay image
+contains all the text). For coloring to work, the images must be white where
+color is desired.
+
+To build this example, the images used look like this:  
+![Images Used](img/dashboard-visual-instructions.png)
```

Comment:
Rereading this writeup, I realize most of my questions are answered. My main question revolves around these images. They are jaggedy on the edges (the pixels, oh the pixels!) and don't really match the polished feel of the rest of the platform, or your blocks, frankly. Is this particular image a requirement? I just wonder if we have some flexibility in how to present the example image, and if we could maybe have one of our designers suggest a cleaner approach.